src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1539,6 +1549,68 @@ static bool removeEntity
 
 /* ****************************************************************************
 *
+* searchContextProviders -
+*
+*/
+void searchContextProviders(const std::string&              tenant,
+                            const std::vector<std::string>& servicePathV,
+                            EntityId&                       en,
+                            ContextAttributeVector&         caV,
+                            ContextElementResponse*         cerP)
+{
+
+  ContextRegistrationResponseVector  crrV;
+  EntityIdVector                     enV;
+  AttributeList                      attrL;
+  std::string                        err;
+
+  /* Fill input data for registrationsQuery() */
+  enV.push_back(&en);
+  for (unsigned int ix = 0; ix < caV.size(); ++ix)
+  {
+    attrL.push_back(caV.get(ix)->name);
+  }
+
+  /* First CPr lookup (in the case some CER is not found): looking in E-A registrations */
+  if (someContextElementNotFound(*cerP))
```

Comment:
Is the way of using the `cerP` argument correct (with `*cerP` in this case) in the function? I think so, but I prefer a double-check :)